homeassistant/components/scrape/translations/en.json

```diff
@@ -1,73 +1,8 @@
 {
-    "config": {
```

Comment:
Can you please move "cleanup config-flow" to a separate PR?

Either it has been decided to never implement it, in which case the cleanup should be done in dedicated PR.
Or it has been decided to implement it "later", in which case the strings shouldn't be removed.

Reply:
Took me a while to understand what you meant.
The strings was never removed when the implementation of the config flow was reverted.
I will reintroduce them as they are not part of this PR e.g. no change for now.
Will try to have it done by tomorrow and then we can add back the config flow as a new PR next after this.